core/embed/rust/src/micropython/buffer.rs

```diff
@@ -71,6 +105,7 @@ impl TryFrom<Obj> for StrBuffer {
             let new = Self {
                 ptr: bufinfo.buf as _,
                 len: bufinfo.len as _,
```

Comment:
this will need a check that `len` fits

Reply:
61244340b4d6a50adecb3e7c3313fb2fce53ffa3